### ISSUE TITLE
[Bug] droidcamclient 6.4.3

### ISSUE THREAD

**Issue by traschke** [RalfEs73/chocolatey-packages#4]
**Please describe the issue.**
The checksum for *droidcamclient 6.4.3* is not correct.

After checking the downloaded exe from chocolatey and additionally downloading it from the official website, for me the SHA256 checksum is `8e71eab54503389698604d796c18a3ed9b1b7a67817a7fad74e86fb1b52ee500`.

**Log**
```
You have droidcamclient v6.4.2 installed. Version 6.4.3 is available based on your source(s).
ERROR: Checksum for 'C:\Users\username\AppData\Local\Temp\chocolatey\droidcamclient\6.4.3\DroidCam.Setup.6.4.3.exe' did not meet 'C70CE7A36E8B81D061E720FF45CD94E59C232E1A6E0AD421B1DD8004853AB68A' for checksum type 'sha256'. Consider passing the actual checksums through with --checksum --checksum64 once you validate the checksums are appropriate. A less secure option is to pass --ignore-checksums if necessary.
The upgrade of droidcamclient was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\droidcamclient\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Failures
 - droidcamclient (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\droidcamclient\tools\chocolateyinstall.ps1'.
 See log for details.
```

**Comment by RalfEs73** [comment]
Thanks for your support. I fixed it in version 6.4.3.20210506

**Comment by RalfEs73** [comment]
Thanks for your support. I fixed it with in version 6.4.3.20210506
